docs/reference/monitoring/configuring-filebeat.asciidoc

```diff
@@ -0,0 +1,190 @@
+[role="xpack"]
+[testenv="basic"]
+[[configuring-filebeat]]
+=== Collecting {es} log data with {filebeat}
+
+[subs="attributes"]
+++++
+<titleabbrev>Collecting log data with {filebeat}</titleabbrev>
+++++
+
+You can use {filebeat} to monitor the {es} log files, collect log events, and
+ship them to the monitoring cluster. In 7.2 and later, your recent logs are
+visible on the *Monitoring* page in {kib}.
+
+//NOTE: The tagged regions are re-used in the Stack Overview.
+
+. Verify that {es} is running and that the monitoring cluster is ready to
+receive data from {filebeat}.
++
+--
+TIP: In production environments, we strongly recommend using a separate cluster 
+(referred to as the _monitoring cluster_) to store the data. Using a separate 
+monitoring cluster prevents production cluster outages from impacting your 
+ability to access your monitoring data. It also prevents monitoring activities 
+from impacting the performance of your production cluster. See
+{stack-ov}/monitoring-production.html[Monitoring in a production environment].
+
+--
+
+. Enable the collection of monitoring data on your cluster.
++
+--
+//include::configuring-metricbeat.asciidoc[tag=enable-collection]
+Set `xpack.monitoring.collection.enabled` to `true` on each node in the
+production cluster. By default, it is is disabled (`false`). 
+
+NOTE: You can specify this setting in either the `elasticsearch.yml` on each 
+node or across the cluster as a dynamic cluster setting. If {es} 
+{security-features} are enabled, you must have `monitor` cluster privileges to 
+view the cluster settings and `manage` cluster privileges to change them.
+
+For example, you can use the following APIs to review and change this setting:
+
+[source,js]
+----------------------------------
+GET _cluster/settings
+
+PUT _cluster/settings
+{
+  "persistent": {
+    "xpack.monitoring.collection.enabled": true
+  }
+}
+----------------------------------
+// CONSOLE 
+For more information, see <<monitoring-settings>> and <<cluster-update-settings>>.
+--
+
+. Optional: Collect metrics about {es}. See <<configuring-metricbeat>> or
+<<collecting-monitoring-data>>.
+
+. Identify which logs you want to monitor. 
```

Comment:
Should we include any guidance on whether to point FIlebeat at unstructured versus structured ES logs?

Reply:
> One thing to consider is that technically users can configure filebeat to the production cluster and setup CCS and the monitoring UI will be able to read those indices. Do we want to include this in the docs?

@chrisronline Unless I'm mistaken, this seems like a configuration method that is broader than just the Filebeat scenario. Therefore, I've created https://github.com/elastic/stack-docs/issues/378 to follow up on those instructions.

Reply:
To be more clear, this is about whether or not we should explicitly tell users to use the `*.json` logs versus the free-text logs. However, I am not sure if there is an advantage from the Filebeat perspective. @ycombinator do you know?

Reply:
Good point, @cachedout.

We should **require** users to use `*.json` logs (as opposed to plaintext logs). The JSON logs are the only ones guaranteed to contain the `cluster_uuid` field, without which the log lines won't be shown in the correct context in the Stack Monitoring UI.

Reply:
Thanks for the clarification!  I've added tips to the steps related to choosing the logs and configuring the filebeat module.